.github/PULL_REQUEST_TEMPLATE/marker.md

```diff
@@ -0,0 +1,55 @@
+---
+When adding a new marker, you must do 2 things:
+- Add an image to show alongside the marker. This image must be stored under `public/photos`, in a folder named after your GitHub username
+- Edit the `places.ts` file under `data`, adding an entry with the following shape:
+```ts
+{
+    author: '...', // First and Last name of the people involved
+    username: '...', // username or usernames of the people involved
+    type: ..., // type of the marker, see below
+    city: '...', // the city where the image was taken
+    country: '...', // the country where the image was taken
+    coordinates: [..., ...], // The coordinates of the marker (ideally, but not necessarily, exact)
+    date: '...', // The date when the image was taken, in the formate YYYY-MM-DD
+    streetview?: '...', // Optional link to Google Map Street View
+    photo: '...' // The actual image taken, stored under "public/photos"
+}
+```
+
+Available pin types are:
+- Sticker
+- Merch
+- Picture
+- Missing
+  
+> Note: Missing should be used when updating a marker
+
+More information is available in this repository's [HOWTO](https://github.com/cesium/CeSIUMpeloMundo/blob/main/HOWTO.md)
+---
+
+> More information is present in this repository's [HOWTO](https://github.com/cesium/CeSIUMpeloMundo/blob/main/HOWTO.md). This is just so the maintainers have the information immediately available
```

Comment:
These phrases are kinda repeated 😅

Reply:
I added this in but then realized that it was just repeating the HOWTO document. They are inside guards (`---`) so they _should_ not appear in the template, but if you still want to I'll delete them.